Mathlib/Order/SuccPred/Limit.lean

```diff
@@ -627,30 +627,30 @@ section isSuccPrelimitRecOn
 section PartialOrder
 
 variable [PartialOrder α] [SuccOrder α]
-  (hs : ∀ a, ¬ IsMax a → C (succ a)) (hl : ∀ a, IsSuccPrelimit a → C a)
+  (succ : ∀ a, ¬IsMax a → motive (succ a)) (isSuccPrelimit : ∀ a, IsSuccPrelimit a → motive a)
```

Comment:
Maybe @vihdzp or @YaelDillies have opinions on whether `isSuccPrelimit` is a good name; some other options are:
* ``of_isSuccPrelimit``
* `limit`

Reply:
I think `limit` is best

Reply:
@eric-wieser OK I have changed the branch names to `limit` (in the `SetTheory` inductors as well) per @YaelDillies's suggestion.

Reply:
@YaelDillies, I've just realized that it's inconsistent to have `(isMax : ∀ a, IsMax a → motive a)` and `(limit : ∀ a, IsSuccPrelimit a → motive a)` in the same induction principle. Perhaps @Parcly-Taxel's original names were best after all?

Reply:
Alternatively, perhaps `isMax` should be renamed to `max`...

Reply:
I don't hugely mind either way

Reply:
Let's save @Parcly-Taxel some work and just revert the last two commits, going back to `isSuccPrelimit` etc?

Reply:
@eric-wieser OK, done.